extensions/mds-logginghouse-binder/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Missing documentation:
- What is this?
- Why is it required?

Reply:
Added README.